src/bci_build/package/appcontainers.py

```diff
@@ -642,3 +642,30 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+REDIS_CONTAINERS = [
+    ApplicationStackContainer(
+        name="redis",
+        pretty_name="Persistent key-value database",
```

Comment:
I would suggest to just call this redis. Otherwise the string `Persistent key-value database` appears in many places (probably also on the registry…) and would just lead to confusion. Redis would be much clearer.
```suggestion
        pretty_name="Redis",
```